src/components/interactive-builder/modals/question/question-form/question/question.test.tsx

```diff
@@ -0,0 +1,197 @@
+import React from 'react';
+import { render, screen, fireEvent, waitFor } from '@testing-library/react';
+import Question from './question.component';
+import { FormFieldProvider } from '../../form-field-context';
+import { useTranslation } from 'react-i18next';
+import RenderTypeComponent from '../rendering-types/rendering-type.component';
+import QuestionTypeComponent from '../question-types/question-type.component';
+import RequiredLabel from '../common/required-label/required-label.component';
+import type { FormField } from '@openmrs/esm-form-engine-lib';
+
+jest.mock('../rendering-types/rendering-type.component', () => ({
+  __esModule: true,
+  default: jest.fn(() => <div data-testid="render-type-component" />),
+}));
+
+jest.mock('../question-types/question-type.component', () => ({
+  __esModule: true,
+  default: jest.fn(() => <div data-testid="question-type-component" />),
+}));
+
+jest.mock('../common/required-label/required-label.component', () => ({
+  __esModule: true,
+  default: jest.fn(({ isRequired, text }) => (
+    <div data-testid="required-label">
+      {text} {isRequired ? '(Required)' : ''}
+    </div>
+  )),
+}));
```

Comment:
Why are these internal components mocked? We should try to avoid mocking internal components like these as much as possible.